siriuspy/siriuspy/clientconfigdb/pvsconfig.py

```diff
@@ -9,6 +9,8 @@
 
 _TIMEOUT = 0.5
 
+# NOTE: this module has become deprecated
```

Comment:
@xresende why?

Reply:
Oops, I searched PVsConfig in `siriuspy`, thinking that all ramp algorithms had been moved to this repository and forgot to search in `hla`. Now I see that the class is being used there. I'll remove the comment. Thanks!

Reply:
done.